openshift/openshift.go

```
@@ -417,3 +417,244 @@ func getSecretReferencesServiceAccount(u unstructured.Unstructured) []v1.ObjectR
+	checkContainerChanges := func(containerType string) {
+		origContainers, _, _ := unstructured.NestedSlice(u.Object, append(basePath, containerType)...)
+		modContainers, _, _ := unstructured.NestedSlice(modified.Object, append(basePath, containerType)...)
+
+		for i := 0; i < len(origContainers) && i < len(modContainers); i++ {
+			origC, ok1 := origContainers[i].(map[string]interface{})
+			modC, ok2 := modContainers[i].(map[string]interface{})
+			if !ok1 || !ok2 {
+				continue
+			}
+
+			origSC, ok1 := origC["securityContext"].(map[string]interface{})
```

golangci-lint (2.12.2)

[error] ineffectual assignment to ok1

(ineffassign)